inference-engine/tests/functional/shared_test_classes/src/single_layer/embedding_segments_sum.cpp

```diff
@@ -3,8 +3,10 @@
 //
 
 #include "shared_test_classes/single_layer/embedding_segments_sum.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
```

Comment:
in general for tests will be a plus to add possibility to create `indices` `offsets` etc as parameter

Reply:
all EmbeddingSegmentsSum operation input arguments have been parameterized already

Reply:
I mean that will be a plus if we can create `indices` and `offsets` inputs as `Parameter` not only as `Constant`

Reply:
discussed offline: will be added later: eshoguli:es/cpu/embedding_bag_sum_test (WIP)